libs/GraphicECS/SFML/Resources/MusicResource.cpp

```diff
@@ -0,0 +1,20 @@
+/*
+** EPITECH PROJECT, 2022
+** R-Type
+** File description:
+** MusicResource
+*/
+
+#include "MusicResource.hpp"
+
+using namespace graphicECS::SFML::Resources;
+
+void MusicResource::addMusic(const musicName_e music_e, const std::filesystem::path &musicPath)
+{
+    std::shared_ptr<sf::Music> music = std::make_shared<sf::Music>();
```

Comment:
Lock the resource before making any changes

Reply:
don't need to lock in the resource because it's at it creation

Reply:
Why must they be created at initialization?